crates/runtime/src/lib.rs

```diff
@@ -141,3 +144,29 @@ pub fn default_wasi() -> Result<WasiCtx> {
     let ctx: WasiCtxBuilder = WasiCtxBuilder::new().inherit_stdio().inherit_args()?;
     Ok(ctx.build())
 }
+
+#[cfg(test)]
+mod unittest {
+    use std::collections::HashMap;
+
+    use slight_common::WasmtimeBuildable;
+    use slight_kv::Kv;
+
+    use crate::Builder;
+
+    #[tokio::test]
+    async fn test_builder_build() -> anyhow::Result<()> {
+        let module = "./test/kv-test.wasm";
+        assert!(std::path::Path::new(module).exists());
+        let mut builder = Builder::from_module(module)?;
+        let kv = slight_kv::Kv::new("kv.filesystem".to_string(), HashMap::default());
+
+        builder
+            .link_wasi()?
+            .link_capability::<Kv>()?
+            .add_to_builder("kv".to_string(), kv);
+
+        let (_, _) = builder.build().await;
+        Ok(())
+    }
+}
```

Comment:
I think it might make sense to move this entire test somewhere else and remove the dependency on `slight_kv` as well. What are your thoughts?

Reply:
The dependency on `slight-kv` is only in `dev-dependencies`, so when we release the crate with `--release` mode, it won't actually be compiled to the package. Thus I am okay with having a slight-kv dev-dependency for testing purposes.